resources/views/registerProject.blade.php

```diff
@@ -0,0 +1,218 @@
+@extends('layouts.app')
+
+@section('title', 'Add Project')
+
+@section('content')
+<!-- Internal CSS stylesheet -->
+<style>
+	#uploadImage{
+		margin-top: 1em;
+		margin-bottom: 2em;
+	}
+
+	.hidden{
+		display: none;
+	}
+</style>
+<div class="ui container" style="margin: 4% auto 8%">
```

Comment:
Move inline styles to resources/sass/app.scss whenever possible. Also, see the styles already available in that file. you want to use consistent styles with other pages (see for example .padded.content style)